### PR TITLE
styling changes, translation render option

### PR DESCRIPTION
**Pull request by SanjeevLakhwani**
- Changed color of counts
- Removed alignment issue with experiment icon
- created an environment variable for enabling/disabling translation

### CODE REVIEW

File: src/js/components/Overview/MakeChartCard.js [hunk 1]
```diff
@@ -50,7 +51,13 @@ const MakeChartCard = ({ section, chart, onRemoveChart }) => {
   return (
     <div key={name} style={{ height: '100%', width: '430px' }}>
       <Card title={t(title)} style={CARD_STYLE} size="small" extra={<Space size="small">{ed}</Space>}>
-        <Chart chartType={chartType} data={data} units={config?.units || undefined} />
+        {data.filter((e) => !(e.x === 'missing')).length !== 0 ? (
+          <Chart chartType={chartType} data={data} units={config?.units || undefined} />
+        ) : (
+          <Row style={{ height: '350px ' }} justify="center" align="middle">
+            <CustomEmpty text="No Data" />
```
Comment (davidlougheed) [inline src/js/components/Overview/MakeChartCard.js:58]: need to add translation term for `No Data`

**Review by davidlougheed — CHANGES_REQUESTED** [review]

File: src/js/components/Overview/MakeChartCard.js (hunk 1 above)
Comment (davidlougheed) [reply src/js/components/Overview/MakeChartCard.js:58]: oh wait maybe no results

Comment (SanjeevLakhwani) [reply src/js/components/Overview/MakeChartCard.js:58]: done

File: src/main.go [hunk 2]
```diff
@@ -32,6 +32,7 @@ type BentoConfig struct {
 	MaxQueryParameters int    `envconfig:"BENTO_PUBLIC_MAX_QUERY_PARAMETERS"`
 	BentoPortalUrl     string `envconfig:"BENTO_PUBLIC_PORTAL_URL"`
 	Port               int    `envconfig:"INTERNAL_PORT" default:"8090"`
+	translated         bool   `envconfig:"TRANSLATED" default:"true"`
```
Comment (brouillette) [inline src/main.go:35]: `translated` should be `Translated` (capital T) so it can be exposed as a public variable to the rest of the code

**Review by brouillette — CHANGES_REQUESTED** [review]
minor change request for golang stuff
LGTM otherwise!

**Review by brouillette — APPROVED** [review]
👌

**Review by davidlougheed — APPROVED** [review]

File: src/js/components/Search/SearchResults.js [hunk 3]
```diff
@@ -32,29 +32,30 @@ const SearchResults = () => {
...
+        loading={isFetchingData}
+      >
         <Row gutter={16}>
           <Col span={4}>
             <Space direction="vertical" size="middle" style={{ display: 'flex' }}>
               <Statistic
                 title={t('Individuals')}
                 value={isValid ? (status === 'count' ? count : t(message)) : '----'}
-                valueStyle={{ color: '#1890ff' }}
+                valueStyle={{ color: '#75787a' }}
```
Comment (v-rocheleau) [inline src/js/components/Search/SearchResults.js:45]: This color value is used as a literal in 4 different places, we could put it in a constant and reference it instead.

**Review by v-rocheleau — APPROVED** [review]

File: src/js/components/Search/SearchResults.js (hunk 3 above)
Comment (davidlougheed) [reply src/js/components/Search/SearchResults.js:45]: ah good point, 2nd this

Comment (SanjeevLakhwani) [reply src/js/components/Search/SearchResults.js:45]: would be in the new pr oops, I have made the change in my current branch so whenever I merge it should be in it

**Pull request merged**